persistent/Database/Persist/SqlBackend.hs

```diff
@@ -6,35 +6,45 @@ module Database.Persist.SqlBackend
       SqlBackend
     , mkSqlBackend
     , MkSqlBackendArgs(..)
+    , SqlBackendHooks(..)
```

Comment:
```suggestion
    , SqlBackendHooks
```

really would like to minimize additions to the public API that require breaking changes to modify (eg `data` constructor direct exposure)